src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -5163,6 +5163,23 @@ void MacroAssembler::verify_cross_modify_fence_not_required() {
 }
 #endif
 
+void MacroAssembler::spin_wait() {
+  for (int i = 0; i < VM_Version::spin_wait_desc().inst_count(); ++i) {
+    switch (VM_Version::spin_wait_desc().inst()) {
+      case SpinWait::NOP:
+        nop();
+        break;
+      case SpinWait::ISB:
+        isb();
+        break;
+      case SpinWait::YIELD:
+        yield();
+        break;
+      default:
+        ShouldNotReachHere();
+    }
+  }
+}
```

Comment:
This merge seems to have gone wrong. Please check to make sure that the `OnSpinWait` changes aren't part of your PR.